src/data/mongo/collections/device.test.ts

```diff
@@ -67,10 +67,10 @@ describe('MongoDeviceTable', () => {
   it('should delete document by name and tenantId', async () => {
```

Comment:
should this test name be updated to delete document by  guid instead of name?

Reply:
nice catch. fixed.